addons/web/static/src/js/views/kanban/search_panel.js

```diff
@@ -46,6 +48,7 @@ var SearchPanel = Widget.extend({
         this.fields = params.fields;
         this.model = params.model;
         this.searchDomain = params.searchDomain;
+        this.preselectFirstRecord = params.preselectFirstRecord || false;
```

Comment:
could be dropped if that becomes an option, and options are obtained with `this.options = options || {};`